### PR TITLE
[BPK-3002]: Update buttons to new brand

### PR DESCRIPTION
**Pull request by ojcurtis**
Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: packages/bpk-mixins/src/mixins/_buttons.scss [hunk 1]
```diff
@@ -337,6 +337,7 @@
 ///   }
 
 @mixin bpk-button--featured {
+  background-image: linear-gradient(-180deg, $bpk-button-featured-gradient-start-color 0%, $bpk-button-featured-gradient-end-color 100%);
```
Comment (georgegillams) [inline packages/bpk-mixins/src/mixins/_buttons.scss]: Can we deprecate `bpk-button-featured-gradient-start-color` and `bpk-button-featured-gradient-end-color`, and create a new token for `bpk-button-featured-color`

File: packages/bpk-tokens/src/web/base/buttons.json [hunk 2]
```diff
@@ -18,41 +18,50 @@
...
+      "deprecated": true
+    },
     "BUTTON_BACKGROUND_COLOR": {
-      "value": "{!GREEN_500}",
+      "value": "{!MONTEVERDE}",
       "type": "color"
     },
     "BUTTON_BACKGROUND_IMAGE": {
-      "value": "linear-gradient(-180deg, {!GREEN_500} 0%, {!GREEN_600} 100%)",
+      "value": "linear-gradient(-180deg, {!MONTEVERDE} 0%, {!MONTEVERDE} 100%)",
```
Comment (georgegillams) [inline packages/bpk-tokens/src/web/base/buttons.json]:
```suggestion
      "value": "{!MONTEVERDE}",
```

File: packages/bpk-tokens/src/web/base/buttons.json [hunk 3]
```diff
@@ -100,7 +109,7 @@
       "type": "color"
     },
     "BUTTON_FONT_SIZE": {
-      "value": "{!FONT_SIZE_BASE}",
+      "value": "1.1875rem ",
```
Comment (georgegillams) [inline packages/bpk-tokens/src/web/base/buttons.json]: We should probably create an alias for this. Something like `BUTTON_FONT_SIZE`

Comment (georgegillams) [reply packages/bpk-tokens/src/web/base/buttons.json]: Same for the other values in this file...

Comment (ojcurtis) [reply packages/bpk-tokens/src/web/base/buttons.json]: Good shout!

File: packages/bpk-tokens/src/web/base/buttons.json [hunk 4]
```diff
@@ -335,23 +344,23 @@
       "type": "box-shadow"
     },
     "BUTTON_SELECTED_BACKGROUND_COLOR": {
-      "value": "{!BLUE_500}",
+      "value": "{!SKY_BLUE}",
       "type": "color"
     },
     "BUTTON_SELECTED_BACKGROUND_IMAGE": {
-      "value": "linear-gradient(-180deg, {!BLUE_600} 0%, {!BLUE_700} 100%)",
+      "value": "linear-gradient(-180deg, {!SKY_BLUE_SHADE_01} 0%, {!SKY_BLUE_SHADE_01} 100%)",
```
Comment (georgegillams) [inline packages/bpk-tokens/src/web/base/buttons.json]:
```suggestion
      "value": "{!SKY_BLUE_SHADE_01}",
```

File: packages/bpk-tokens/src/web/base/buttons.json [hunk 5]
```diff
@@ -387,20 +396,20 @@
       "type": "box-shadow"
     },
     "BUTTON_FEATURED_BACKGROUND_IMAGE": {
-      "value": "linear-gradient(-180deg, {!PINK_500} 0%, {!PINK_600} 100%)",
+      "value": "linear-gradient(-180deg, {!SKY_BLUE} 0%, {!SKY_BLUE} 100%)",
```
Comment (georgegillams) [inline packages/bpk-tokens/src/web/base/buttons.json]:
```suggestion
      "value": "{!SKY_BLUE}",
```

File: packages/bpk-tokens/src/web/base/buttons.json (hunk 3 above)
Comment (georgegillams) [inline packages/bpk-tokens/src/web/base/buttons.json]:
```suggestion
      "value": "!{BUTTON_FONT_SIZE}",
```

**Pull request merged**